inductiva/resources/machine_cluster.py

```diff
@@ -38,6 +38,9 @@ def __init__(self,
                          data_disk_gb=data_disk_gb,
                          register=register)
         self.num_machines = num_machines
+        #Number of active machines at the time of
+        #the request machine_groups.get()
+        self._requested_machines = 0
```

Comment:
I am getting confused...

We set `num_machines`, which is technically the requested number of machines... Then, you establish a variable that is named `_requested_machines`, which technically is the actual number of machines requested. So it makes no sense, on the api_response to fetch the active number of machines to `num_machines`. 

So, to make it simpler, we should have a variable `num_machines` that always refers to the number of machines requested and can be used to reinitialize the class back again, from the number of `max_vms`. Then, you can have the `_active_machines` variable that receives from the backend the `num_vms` which refers to the active machines. But in this way, there are no conflicts, since the class created from the response is exactly the same. 

I think the names coming from the backend should also be formatted to address this, but we can deal with that after a more general refactor.

Reply:
The way it is now, ´num_machines´ is doing exactly the same as it was before the change. I don't want to change it because I am afraid it will break something somewhere.
So, by keeping ´num_machines´ the exact same, I hope to introduce fewer points of failure.

Reply:
It is in the object initialization, but not in the `api_response` where it seems to be receiving the `num_vms` which is the active machines, no?

Reply:
> It is in the object initialization, but not in the `api_response` where it seems to be receiving the `num_vms` which is the active machines, no?

Yes, but previously `num_vms` was also the active machines. Sometimes we had 0 machines on a machine group cause the machines were not active yet.

Reply:
I think it was until @begaiym-k changed it in the backend, because last week or so we weren't even capturing this.